src/metpy/calc/thermo.py

```diff
@@ -25,7 +25,7 @@
 @exporter.export
 @preprocess_and_wrap(wrap_like='temperature', broadcast=('temperature', 'dewpoint'))
 @check_units('[temperature]', '[temperature]')
-def relative_humidity_from_dewpoint(temperature, dewpoint):
+def relative_humidity_from_dewpoint(temperature, dewpoint, phase='liquid'):
```

Comment:
You'll need to add this to the documentation.

I don't know maintainer policy on magic string constants vs. instances of `enum.Enum`, though the latter would aid discoverability